lib/modules/datasource/packagist/index.ts

```diff
@@ -49,7 +60,7 @@ export class PackagistDatasource extends Datasource {
 
   private static getPackagistFileUrl(
     regUrl: string,
-    regFile: RegistryFile
+    regFile: schema.RegistryFile
```

Comment:
can we import the types directly instead of namespacing them? i don't like `import * as ...`.

would also cause less changes